components/InfoPane/InfoPane.tsx

```diff
@@ -27,6 +27,7 @@ export default function InfoPane(props: HTMLProps<HTMLDivElement>) {
             package.json file
           </QueryLink>
         </li>
+        <li>package.json pastes (copied as text or file)</li>
       </ul>
 
       <FileUploadControl />
```

Comment:
And then, after:

- #240


The drop area will also be turned into two bullet points on the list above:

> - package.json drag and drop anywhere on the page
> - package.json [picked](#select) from your file system